specifyweb/frontend/js_src/lib/utils/utils.ts

```diff
@@ -403,7 +404,7 @@ export function throttle<ARGUMENTS extends RA<unknown>>(
   function later(): void {
     previousTimestamp = Date.now();
     timeout = undefined;
-    callback(...previousArguments!);
+    callback.bind(thisArgument)(...previousArguments!);
```

Comment:
See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/apply and https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/call

Reply:
> this.updateCellInfoStats = throttle(this.updateCellInfoStats.bind(this), throttleRate);
I was going to do this, but I figured that writing the bind in the throttle function would prevent future bugs like this.

Reply:
I would have preferred to use apply rather than bind, but Typescript did not like me for it. 
Typescript expects the arguments parameter in `function.apply(args)` to always be something like `RA<unkown>`, while the function in this case expects its arguments to match the type signatures of the definition (in this case the ARGUMENTS type). 

Even though the ARGUMENTS type definition is `extends RA<unknown>`, I was not able to make Typescript agree on a proper type